src/include/raft.h

```diff
@@ -244,6 +246,7 @@ struct raft_entry_header
     uint32_t         reh_entry_sz[RAFT_ENTRY_NUM_ENTRIES];
     uint8_t          reh_leader_change_marker; // noop
     uint8_t          reh_num_entries; // number of raft entries
+    uint64_t         reh_apply_handler_version; // Version of handler to use
```

Comment:
Please place this above the `uint8_t`'s so that the struct contains no internal padding.